### PR TITLE
deno upgraded to version 2 and express-deno added

### PR DESCRIPTION
**Pull request by zahidcakici**
Deno upgraded to version 2.0 and with the new feature npm packages are now available like express.js.

`express-deno` added to the project

### CODE REVIEW

**Comment by zahidcakici** [comment]
For the failing test: [opine framework](https://github.com/cmorten/opine) cut its support (last release 16-04-2023) and since it is an express port, recommends using express npm package with deno 2 just as the PR I opened.  

We might consider removing it.

Screenshot 2024-10-11 at 16 07 02

**Comment by waghanza** [comment]
Could you ping the author of `express-deno` author, or any core contributor  @zahidcakici ?

We should remove `opine` @cmorten

File: javascript/config.yaml [hunk 1]
```diff
@@ -35,7 +35,7 @@ language:
         - npm
       bootstrap:
         - npm --location=global install pm2
-      command: nohup pm2-runtime start "deno run --allow-net --allow-read=. app.ts" -i max
+      command: nohup pm2-runtime start "deno run --allow-env --allow-net --allow-read=. app.ts" -i max
```
Comment (waghanza) [inline javascript/config.yaml:38]: could you please explain this change ?

Comment (zahidcakici) [reply javascript/config.yaml]: I was getting an error, I guess deno 2 with npm:express requires `--allow-env` flag.

I would love to contribute on clustering deno btw

**Review by zahidcakici — COMMENTED** [review]
Yep, after the failing task again I'm pretty sure we need that flag

**Review by waghanza — APPROVED** [review]

**Comment by waghanza** [comment]
@zahidcakici results has been updated https://web-frameworks-benchmark.netlify.app/result?f=express,express-deno

**Pull request merged**